### PR TITLE
Updated cob4-3 config

### PR DESCRIPTION
**Pull request by mathias-luedtke**
- ignore BMS in diagnostics aggregator (**should be reverted later**)
- base calibration
- read currents and include them in joint_states
- stuck detector (needs https://github.com/ipa320/cob_control/pull/111)
- **do not respawn BMS driver**

### CODE REVIEW

File: cob_bringup/robots/cob4-3.xml [hunk 1]
```diff
@@ -31,6 +31,10 @@
 		<include file="$(find cob_bringup)/controllers/base_controller_plugin.launch">
 			<arg name="robot" value="$(arg robot)"/>
 		</include>
+		<include file="$(find cob_omni_drive_controller)/launch/stuck_detector.launch">
```
Comment (NadiaHG) [inline cob_bringup/robots/cob4-3.xml:34]: please add the `cob_omni_drive_controller` dependency  to the package.xml

**Comment by floweisshardt** [comment]
https://github.com/ipa320/cob_control/pull/111#issuecomment-235715133 says that we can handle the stuck detection on the elmo firmware, which I prefer over handling it in ROS.
So please remove all stuck detection related modifications from this PR.

File: cob_bringup/robots/cob4-3.xml (hunk 1 above)
Comment (mathias-luedtke) [inline cob_bringup/robots/cob4-3.xml:34]: I wasn't aware that cob_omni_drive_controller was pulled in via cob_hardware_config. Made it explicit now.

**Comment by mathias-luedtke** [comment]
> ipa320/cob_control#111 (comment) says that we can handle the stuck detection on the elmo firmware, which I prefer over handling it in ROS.

Me too, but it did not work out. Elmos can detect stuck errors (in velocity mode) only by analyzing the current. The high level controller can verify against the actual collision target.

**Comment by floweisshardt** [comment]
FR: OK

**Comment by NadiaHG** [comment]
FR: OK

**Pull request merged**